src/node/install.sh

```diff
@@ -227,7 +227,7 @@ install_yarn() {
                 # Yum/DNF want to install nodejs dependencies, we'll use NPM to install yarn
                 su ${USERNAME} -c "umask 0002 && . '${NVM_DIR}/nvm.sh' && nvm use ${_ver} && npm install --global yarn"
```

Comment:
Can you help bump the patch version of this Feature? 

https://github.com/devcontainers/features/blob/a9247d8cdb3388d8849cf13ef5b0f5da862a84e7/src/node/devcontainer-feature.json#L3

Reply:
Updated the version to `1.6.1`